### PR TITLE
Fix: tests failing on travis

### PR DESCRIPTION
**Pull request by zebateira**
This is a short term solution I found to make the tests pass in Travis: increase the timeouts.
The reason why some tests fail seems to be because submitting the code solution takes some time. up to 30 seconds sometimes, but it can spike further - 60s timeout was not enough in some rare occasions.
So it seems that some ipfs operations are taking longer than desirable. I'm not sure why this is the case only in travis. Locally (and production) it's not that slow, but I was able to reproduce it running on electron locally.

### CODE REVIEW

File: src/components/Lesson.vue [hunk 1]
```diff
@@ -136,7 +137,7 @@ import Validator from './Validator.vue'
 import TutorialCompletionCallout from './callouts/TutorialCompletion.vue'
 import TypeIcon from './TypeIcon.vue'
 
-const MAX_EXEC_TIMEOUT = 10000
+const MAX_EXEC_TIMEOUT = isProduction ? 10e3 : 60e3
```
Comment (terichadbourne) [inline src/components/Lesson.vue:140]: @zebateira Can you teach me why these aren't numbers? Or is this some fancy number formatting I haven't learned yet?

Comment (terichadbourne) [reply src/components/Lesson.vue:140]: Figured it out but would suggest that these numbers are small enough that it makes sense to write them out to make the code easier for newcomers to follow.

Comment (zebateira) [inline src/components/Lesson.vue:140]:
```suggestion
const MAX_EXEC_TIMEOUT = isProduction ? 10000 : 60000
```

Comment (zebateira) [reply src/components/Lesson.vue]: @terichadbourne changed 👍

**Pull request merged**